susemanager-utils/susemanager-sls/salt/clusters/createcluster.sls

```diff
@@ -0,0 +1,24 @@
+{%- if pillar.get('ssh_auth_sock', False) %}
+mgr_ssh_agent_socket_clusters_createcluster:
+  environ.setenv:
+    - name: SSH_AUTH_SOCK
+    - value: {{ pillar['ssh_auth_sock'] }}
+{%- endif %}
+
+mgr_cluster_create_cluster:
```

Comment:
I assume that if a specific type of cluster does not offer the `init` workflow, we can skip the initialization and jump directly to other operations, correct?

Reply:
That's correct. Or, even in the case of CaaSP, if the cluster has been already created, we can directly operate it with i.a. `clusters.addnode`.